### PR TITLE
Tests for indentby tag

### PR DESCRIPTION
**Pull request by kuter**

### CODE REVIEW

File: django_extensions/templatetags/indent_text.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def indentby(parser, token):
     args = token.split_contents()
     largs = len(args)
     if largs not in (2, 4):
-        raise template.TemplateSyntaxError("%r tag requires 1 or 3 arguments")
+        raise template.TemplateSyntaxError(
```
Comment (trbs) [inline django_extensions/templatetags/indent_text.py]: Could we keep the old version of the intentation please ? :-)
(We have `max-line-length = 512` in `setup.cfg`)

Comment (kuter) [reply django_extensions/templatetags/indent_text.py]: sure we can .. how about change %r into indentby ? tests gives me %r which is not very informative

Comment (trbs) [reply django_extensions/templatetags/indent_text.py]: Think the `%r` is a typo and fully support the change itself 👍

**Comment by trbs** [comment]
awesome ! thanks for all your help !

**Pull request merged**